### PR TITLE
issue #20 - Map component

### PR DESCRIPTION
**Pull request by AmyHong0502**
Implemented Map component and added basic layout with hardcoded values.

There's the `size/XXL` label but it's because I installed a package and added 1360 lines of JSON data.

### CODE REVIEW

File: src/pages/selection/schedule/index.jsx [hunk 1]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+        <div className='col-12'>
+          <SectionHeader
+            title={dummyHeader[2].title}
+            subtitle={dummyHeader[2].subtitle}
+            level='2'
+            description={dummyHeader[2].description}
+          />
+        </div>
+        <div className='col-md-6'>
+          <i class='far fa-calendar-check' />
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 2]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+          <select name='schedule'>
+            <option value='May 12, 2019'>May 12, 2019</option>
+            <option value='May 13, 2019'>May 13, 2019</option>
+            <option value='May 14, 2019'>May 14, 2019</option>
+            <option value='May 15, 2019'>May 15, 2019</option>
+          </select>
+          <br />
+          <i class='fas fa-map-marker-alt' />
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 3]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+          <input name='location' type='text' />
+          <Map />
+        </div>
+        <div className='col-md-6'>
+          <ul class='list-group list-group-flush'>
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 4]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+            <li class='list-group-item'>
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className

File: src/pages/selection/schedule/index.jsx [hunk 5]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+              <div>
+                <span className='lead'>Centre Branch</span>
+                <p>350 W Georgia st / 0.4km away</p>
+                <i class='fab fa-accessible-icon' /> Wheelchair Access: via ramp
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 6]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+                at the main entrance on East Hastings St
+                <br />
+                <i class='fas fa-parking' /> Parking: Street
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 7]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+                <br />
+                <i class='fas fa-toilet' /> Washrooms: Wheelchair Accessible
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 8]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+                <br />
+                <i class='fas fa-phone' /> 604-456-7890
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 9]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+                <br />
+                <i class='fas fa-envelope' /> dummy@planyourvote.com
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 10]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+                <br />
+                <i class='far fa-clock' /> Voting Hours:
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 11]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+                <br />
+                May 12: 08:30AM - 05:00PM
+                <br />
+                May 13: 10:00AM - 03:00PM
+              </div>
+            </li>
+            <li class='list-group-item'>Dapibus ac facilisis in</li>
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 12]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+            <li class='list-group-item'>Morbi leo risus</li>
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 13]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+            <li class='list-group-item'>Porta ac consectetur ac</li>
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

File: src/pages/selection/schedule/index.jsx [hunk 14]
```diff
@@ -5,15 +5,59 @@ import dummyHeader from 'constants/dummyData/pages.json';
...
+            <li class='list-group-item'>Vestibulum at eros</li>
```
Comment (Lizhclee) [inline src/pages/selection/schedule/index.jsx]: Please change class to className.

Comment (AmyHong0502) [reply src/pages/selection/schedule/index.jsx]: Oops, fixed it. Thanks!

**Pull request merged**